README.md

```diff
@@ -99,6 +98,45 @@ tuner.run()
 
 The above example runs ASHA with 4 asynchronous workers on a local machine.
 
+## Supported HPO methods
+
+The following hyperparameter optimization (HPO) methods are available in Syne Tune:
+
+Method | Reference | Searcher | Asynchronous? | Multi-fidelity? | Transfer? 
+:--- | :---: | :---: | :---: | :---: | :---: 
+Grid Search | Someone (sometime) | deterministic | no | no | no 
```

Comment:
Could you update the reference "Someone (sometime)"? (we might put Bergstra as the method is described compared there as I am not sure what would be the original reference)

Also, I think we should mark this one as asynchronous given that there is no difficulty for the async case and it is supported in syne tune.

Reply:
Yeah, Bergstra and Bengio seem a default reference, even though they rather bash grid search